docs/book/en/src/dev/rust/wasinn.md

```diff
@@ -0,0 +1,158 @@
+# Neural Network for WASI
+
+In WasmEdge, we implemented the [WASI-NN](https://github.com/WebAssembly/wasi-nn) (Neural Network for WASI) proposal to allows WebAssembly programs to access machine learning(ML) functions, in the fashion of "graph" loader API. [WASI-NN](https://github.com/WebAssembly/wasi-nn) allows users to "load, bind, and evaluate" models by functions:
```

Comment:
`to allow`
`access the Machine Learning (ML) functions with the fashion of graph loader APIs`
`by the following functions`